bayesian/repr_license_finder_go.py

```diff
@@ -0,0 +1,114 @@
+"""Identifies representative license for go eco repo by considering all the dependencies."""
+
+from __future__ import division
+from flask import current_app
+from requests import get, post, exceptions
+import json
+from .utils import GREMLIN_SERVER_URL_REST
+from f8a_worker.utils import get_session_retry
+from .utils import fetch_file_from_github
+import requests
+import logging
+import traceback
+
+
+class GetReprLicense():
+    """Summary.
+
+    input:url:repo url to find repr_license
```

Comment:
This documentation should be moved to `__init__()` instead. Please summarize the overall work done by GetReprLicense here.